src/engine/GroupBy.cpp

```diff
@@ -1019,11 +997,6 @@ void GroupBy::substituteAllAggregates(
   }
 }
 
-// _____________________________________________________________________________
-template <typename A>
-concept SupportedAggregates =
-    ad_utility::SameAsAnyTypeIn<A, GroupBy::AggregationDataVectors>;
-
```

Comment:
I liked them where they were, or do you require them in some other translation unit in the upcoming lazy groupBy PR?

Reply:
Exactly